pynxtools/dataconverter/helpers.py

```diff
@@ -203,6 +205,15 @@ def generate_template_from_nxdl(
         generate_template_from_nxdl(child, template, path, nxdl_root, nxdl_name)
 
 
+def get_all_entry_names(template: Template) -> Set[str]:
+    entry_names = set()
+    for key in template:
+        entry_name_match = re.search(r"\/ENTRY\[([a-zA-Z0-9_\.]+)\]", key)
+        if entry_name_match is not None:
+            entry_names.add(entry_name_match.group(1))
+    return entry_names
```

Comment:
Can we move this to the Template class instead? Makes sense to have it there.

Reply:
Sure. good suggestion, I'll move it